Core/include/Acts/Propagator/ConstrainedStep.hpp

```diff
@@ -59,33 +60,46 @@ class ConstrainedStep {
 
   /// constructor from Scalar
   /// @param value is the user given initial value
-  constexpr explicit ConstrainedStep(Scalar value) { m_values[user] = value; }
+  constexpr explicit ConstrainedStep(Scalar value) { setUser(value); }
 
   /// set accuracy by one Scalar
   ///
   /// this will set only the accuracy, as this is the most
   /// exposed to the Propagator
   ///
   /// @param value is the new accuracy value
-  constexpr void setValue(Scalar value) {
+  constexpr void setAccuracy(Scalar value) {
```

Comment:
Maybe I'm stupit, but how could `actor` and `aborter` values ever be set by this interface?